.gitignore

```diff
@@ -4,6 +4,7 @@
 .DS_Store
 .idea
 .project
+.settings/
```

Comment:
Did the ignore go in after the `.settings/` directory was added? 

Can you remove the files in `.settings/` directory please & the directory itself:

* .settings/org.eclipse.php.core.prefs
* .settings/org.eclipse.php.debug.core.Debug_Process_Preferences.prefs

*Keep `.settings/` in the ignore*